doc/install.md

```diff
@@ -3,200 +3,287 @@
-   k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.9
-   ```
 ### Test Coscheduling
 
-Assume there is 1500m free cpu resource in the cluster.
-
-1. Create a replicaset with `pod-group.scheduling.sigs.k8s.io/name: nginx` and `pod-group.scheduling.sigs.k8s.io/min-available: "4"`. Each pod acquires 500m cpu.
-
-   ```yaml
-   apiVersion: apps/v1
-   kind: ReplicaSet
-   metadata:
-     name: nginx
-     namespace: default
-     labels:
-       app: nginx
-   spec:
-     replicas: 4
-     selector:
-       matchLabels:
-         app: nginx
-     template:
-       metadata:
-         name: nginx
-         labels:
-           app: nginx
-           pod-group.scheduling.sigs.k8s.io/name: nginx
-           pod-group.scheduling.sigs.k8s.io/min-available: "4"
-       spec:
-         containers:
-         - name: nginx
-           image: nginx
-           imagePullPolicy: IfNotPresent
-           resources:
-             limits:
-               cpu: 500m
-             requests:
-               cpu: 500m
-   ```
-
-2. Check pods
-
-   All nginx pods are expected to be `Pending` as they cannot be co-scheduled altogether.
-
-   ```bash
-   $ kubectl get pod -n default
-   NAME          READY   STATUS    RESTARTS   AGE
-   nginx-25797   0/1     Pending   0          2m
-   nginx-6fdcm   0/1     Pending   0          2m
-   nginx-stcxq   0/1     Pending   0          2m
-   nginx-xn25q   0/1     Pending   0          2m
-   ```
-
-## Uninstall Scheduler-plugins
-
-1. Delete ReplicaSet
-   ```bash
-   kubectl delete replicaset -n default nginx
-   ```
-
-2. Recover `kube-scheduler.yaml` and delete `coscheduling-config.yaml`
-
-   * If the cluster is created by `kubeadm` or `minikube`, log into Master node:
-       ```bash
-       mv /etc/kubernetes/kube-scheduler.yaml /etc/kubernetes/manifests/
-       rm /etc/kubernetes/coscheduling-config.yaml
-       ```
-
-   * If the cluster is created by `kind`, enter the Master's container:
-       ```bash
-       sudo docker exec -it $(sudo docker ps | grep control-plane | awk '{print $1}') bash
-       mv /etc/kubernetes/kube-scheduler.yaml /etc/kubernetes/manifests/
-       rm /etc/kubernetes/coscheduling-config.yaml
-       exit
-       ```
-
-4. Check default scheduler state
-
-   ```bash
-   $ kubectl get pod -n kube-system | grep kube-scheduler
-   kube-scheduler-xqcluster-control-plane            1/1     Running   0          91s
+Now, we're able to verify how the coscheduling plugin works.
+
+1. Create a PodGroup custom object called `pg1`:
+
+    ```yaml
+    # podgroup.yaml
+    apiVersion: scheduling.sigs.k8s.io/v1alpha1
+    kind: PodGroup
+    metadata:
+      name: pg1
+    spec:
+      scheduleTimeoutSeconds: 10
+      minMember: 3
+    ```
+
+    ```bash
+    $ kubectl apply -f podgroup.yaml
+    ```
+
+1. Create a deploy labelled `pod-group.scheduling.sigs.k8s.io: pg1` to associated with PodGroup
+   `pg1` created in the previous step.
```

Comment:
always get `1.`, seems not intentional

Reply:
When the list is long and frequently modified, it's not easy to always make its numbers ordered accurately :) So keep all as `1.` and make them indented well, markdown will do the rest things.